IOPool/Streamer/interface/StreamSerializer.h

```diff
@@ -18,77 +18,100 @@
 #include "DataFormats/Provenance/interface/SelectedProducts.h"
 #include "FWCore/Utilities/interface/get_underlying_safe.h"
 
-const int init_size = 1024 * 1024;
+const int init_size = 0;  //will be allocated on first event
+const unsigned int reserve_size = 50000;
```

Comment:
`constexpr`

Reply:
Better to add both inside `struct SerializeDataBuffer` to avoid having these names be in the global namespace.

Reply:
Thanks for comments. I moved them inside this struct.